notebooks/tutorials/beginner_tutorials/train_Resnet_on_HSN.ipynb

```diff
@@ -0,0 +1,1131 @@
+       "          0., 0., 0.],\n",
+       "         [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0., 0.,\n",
+       "          0., 0., 0.],\n",
+       "         [0., 1., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+       "          0., 0., 0.]], dtype=torch.float16)}"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from birdset.datamodule.base_datamodule import DatasetConfig\n",
+    "from birdset.datamodule.birdset_datamodule import BirdSetDataModule\n",
+    "from birdset.datamodule.components.event_decoding import EventDecoding\n",
+    "from birdset.datamodule.components.transforms import PreprocessingConfig, BirdSetTransformsWrapper\n",
+    "from torchaudio.transforms import Spectrogram\n",
+    "\n",
+    "transforms = BirdSetTransformsWrapper(model_type='vision',preprocessing=PreprocessingConfig(spectrogram_conversion= Spectrogram(\n",
+    "            n_fft=1024,\n",
+    "            hop_length=320,\n",
+    "            power=2.0,\n",
+    "        ),), decoding=EventDecoding(sampling_rate=32000), task=\"multilabel\")\n",
+    "# initiate the data module\n",
+    "dm = BirdSetDataModule(\n",
+    "    dataset= DatasetConfig(\n",
+    "        data_dir='../../data_birdset/HSN',\n",
+    "        dataset_name='HSN',\n",
+    "        hf_path='DBD-research-group/BirdSet',\n",
+    "        hf_name='HSN',\n",
+    "        n_classes=21,\n",
+    "        n_workers=3,\n",
+    "        val_split=0.2,\n",
+    "        task=\"multilabel\",\n",
+    "        classlimit=500,\n",
+    "        eventlimit=5,\n",
+    "        sampling_rate=32000,\n",
+    "    ),\n",
+    "    transforms=transforms\n",
+    ")\n",
+    "\n",
+    "   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step2) Prepare Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# prepare the data (download dataset, ...)\n",
+    "dm.prepare_data()\n",
+    "# setup the dataloaders\n",
+    "dm.setup(stage=\"fit\")\n",
+    "# get the dataloaders\n",
+    "train_loader = dm.train_dataloader()\n",
+    "test_loader = dm.test_dataloader()\n",
+    "# get the first batch\n",
+    "batch = next(iter(train_loader))\n",
+    "# get shape of the batch\n",
+    "print(batch[\"input_values\"].shape)\n",
+    "print(batch[\"labels\"].shape)\n",
+    "batch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step3) Prepare trainer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lightning import Trainer \n",
+    "\n",
+    "min_epochs = 1\n",
+    "max_epochs = 20\n",
+    "trainer = Trainer(min_epochs=min_epochs, max_epochs=max_epochs, accelerator=\"gpu\", devices=0)"
```

Comment:
```suggestion
    "trainer = Trainer(min_epochs=min_epochs, max_epochs=max_epochs, accelerator=\"gpu\", devices=[0])"
```
This must have thrown an error. Always run your code before commiting ;)

Reply:
done a70be7bddfe5018a8edd173a0bb6a6948328e9dc